agent/engine/docker_image_manager.go

```diff
@@ -335,7 +335,10 @@ func (imageManager *dockerImageManager) removeNonECSContainers(ctx context.Conte
 	}
 	var nonECSContainerRemoveAvailableIDs []string
 	for _, id := range nonECSContainersIDs {
-		response, _ := imageManager.client.InspectContainer(ctx, id, dockerclient.InspectContainerTimeout)
+		response, icErr := imageManager.client.InspectContainer(ctx, id, dockerclient.InspectContainerTimeout)
+		if icErr != nil {
+			seelog.Errorf("Error inspecting non-ECS container id: %s - %v", id, icErr)
```

Comment:
do we need to add a `continue` in this case since `response` will contain nothing? same below

Reply:
you're correct on both accounts, will update.